pkgs/applications/misc/trenchbroom/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv, fetchFromGitHub
+, cmake, ninja, git, pandoc
+, libGL, libGLU, libXxf86vm, freeimage
+, qtbase, wrapQtAppsHook
+, copyDesktopItems, makeDesktopItem
+}:
+
+let
+  ver = {
+    year = "2021";
+    number = "1";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "TrenchBroom";
+  version = with ver; "${year}.${number}";
```

Comment:
You could probably use ``lib.versions.major/minor`` to parse the git tag. Then auto update r-rynatm/nix-update should work.

Reply:
Done. Thanks @SuperSandro2000